attestation-service/verifier/test_data/cca_evidence_claim_output.txt

```diff
@@ -0,0 +1 @@
+Object {"platform": Object {"cca-platform-challenge": String("AQICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIC"), "cca-platform-sw-components": Array [Object {"measurement-type": String("BL"), "measurement-value": String("AwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwM="), "signer-id": String("BAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQ="), "version": String("3.4.2")}]}, "realm": Object {"cca-realm-extensible-measurements": Array [String("Q0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQw=="), String("Q0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQw=="), String("Q0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQw=="), String("Q0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQw==")], "cca-realm-hash-algo-id": String("sha-256"), "cca-realm-initial-measurement": String("Q0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQ0NDQw=="), "cca-realm-personalization-value": String("QURBREFEQURBREFEQURBREFEQURBREFEQURBREFEQURBREFEQURBREFEQURBREFEQURBREFEQURBREFEQURBRA=="), "cca-realm-public-key-hash-algo-id": String("sha-512")}}
```

Comment:
This file is generated automatically by unit tests in `verifier`. I'd suggest to  add them (together with other `parse_*.txt`) to `.gitignore`